src/css/base/_a-tag.scss

```diff
@@ -0,0 +1,9 @@
+a {
+  text-decoration: none;
+  display: inline-block;
+  width: 100%;
+  height: 100%;
+}
+button {
```

Comment:
File name indicates that it will only have rules for `a` tag. Rules for `button` don't fit here. Either rename the file or move it into a separate file.